### PR TITLE
set logging level to info

### PR DESCRIPTION
**Pull request by gintautassulskus**
https://github.com/GoogleContainerTools/minikube-build-tools-for-java/issues/144

### CODE REVIEW

**Comment by loosebazooka** [comment]
@coollog did we have this at lifecycle before because it was a separate task? `info` does seem like the right level, but I can't remember why it was `lifecycle` before.

**Comment by coollog** [comment]
I believe `lifecycle` was just an arbitrary choice and `info` is more appropriate. `getDockerEnv` is just a method on the extension and not a separate task.

File: minikube-gradle-plugin/src/main/java/com/google/cloud/tools/minikube/util/CommandExecutor.java [hunk 1]
```diff
@@ -16,6 +16,9 @@
 
 package com.google.cloud.tools.minikube.util;
 
+import org.gradle.api.GradleException;
```
Comment (coollog) [inline minikube-gradle-plugin/src/main/java/com/google/cloud/tools/minikube/util/CommandExecutor.java]: It looks like imports got rearranged. Please run `gradle goJF` to format the code with the google java format.

Comment (gintautassulskus) [reply minikube-gradle-plugin/src/main/java/com/google/cloud/tools/minikube/util/CommandExecutor.java]: done

**Comment by coollog** [comment]
Hi @gintautassulskus , looks like a test failed. Running `./gradlew build` should be able to reproduce this:

```
com.google.cloud.tools.minikube.util.CommandExecutorTest > testRunCommandWithLogging_success FAILED
    org.mockito.exceptions.verification.VerificationInOrderFailure: 
    Verification in order failure
    Wanted but not invoked:
    loggerMock.lifecycle("some output line 1");
    -> at com.google.cloud.tools.minikube.util.CommandExecutorTest.testRunCommandWithLogging_success(CommandExecutorTest.java:127)
    Wanted anywhere AFTER following interaction:
    loggerMock.debug(
        "Running command : someCommand someOption"
    );
    -> at com.google.cloud.tools.minikube.util.CommandExecutor.run(CommandExecutor.java:89)
        at com.google.cloud.tools.minikube.util.CommandExecutorTest.testRunCommandWithLogging_success(CommandExecutorTest.java:127)
```

**Comment by gintautassulskus** [comment]
Apologies. Negligence due to rushing things.

**Pull request merged**